dist/lozad.js

```diff
@@ -70,6 +73,7 @@ var lozad = function () {
       var elements = document.querySelectorAll(selector);
       for (var i = 0; i < elements.length; i++) {
         if (isLoaded(elements[i])) {
+          console.log(elements[i]);
```

Comment:
what's this doing here? 😮

Reply:
Sorry! I was just debugging LOL

Reply:
Fixed. I will add the build script to the precommit hook, to prevent that kind of thing to happen.

Reply:
Most importantly, don't forget to build correctly before publishing :)